fcgiwrap.c

```diff
@@ -578,6 +578,7 @@ static void handle_fcgi_request(void)
 			}
 
 			execl(filename, filename, (void *)NULL);
+			fprintf(stderr, "Error: %s\n", strerror(errno));
```

Comment:
I would prefer to keep consistent to the current definition of `cgi_error`, because adding additional argument will make all the cgi_error calling filled with a NULL expect the "Cannot execute script" one. Create log functionality for only one possible error is not useful too. What I am thinking is just make a string containing the `strerror` message and pass it into `filename` argument. Actually I should say, cgi_error is not a very nice design, although it looks simple, but it assume every variable string should append in the end, which is not the case.